mlflow/server/__init__.py

```diff
@@ -134,6 +134,118 @@ def serve_get_logged_model_artifact(model_id: str):
     return get_logged_model_artifact_handler(model_id)
 
 
+# Evaluation Dataset AJAX endpoints
+@app.route(_add_static_prefix("/ajax-api/2.0/mlflow/evaluation-datasets/search"), methods=["POST"])
```

Comment:
```suggestion
@app.route(_add_static_prefix("/ajax-api/3.0/mlflow/evaluation-datasets/search"), methods=["POST"])
```

Can we use 3.0 for new APIs?